src/crimson/net/Socket.h

```diff
@@ -197,27 +197,7 @@ class FixedCPUServerSocket
   FixedCPUServerSocket(const FixedCPUServerSocket&) = delete;
   FixedCPUServerSocket& operator=(const FixedCPUServerSocket&) = delete;
 
-  seastar::future<> listen(entity_addr_t addr) {
```

Comment:
:+1: ... and for better header readability!

Even if there any performance reason (which I doubt about) for keeping the code generally inlineable, we should switch out from in-class definition and put it somewhere at the header's bottom.